ui/raidboss/data/triggers/o9n.js

```diff
@@ -92,9 +91,9 @@
         'Stray Spray': 'Chaosspritzer',
         'Tsunami': 'Tsunami',
         'Umbra Smash': 'Schattenschlag',
+        'Long/Lat Implosion': 'Horizontale/Vertikale Implosion',
 
         // FIXME
-        'Long/Lat Implosion': 'Hz/Vert Implosion',
         '\\(ALL\\)': '(ALL)',
```

Comment:
This line. I've got no clue what to do with that or how to fix that.

Reply:
The "ALL" here is just meant to be "everybody" as compared to "T/H" or "DPS" for explosions.